File: cpp/ciclofor.cpp
#include <iostream>
#include <cmath>  // Para usar la función pow() que calcula la potencia

using namespace std;

int main() {
    int A, C;
    
    // Solicitar el primer valor
    cout << "Ingrese el primer valor: ";
    cin >> A;

    // Inicia la variable B 
    int B = 0;

    // Solicitar el segundo valor
    cout << "Ingrese el segundo valor: ";
    cin >> C;

    // Calcular la potencia de A elevado a C
    int valor = pow(A, C);  // 'pow' devuelve el valor de A elevado a C

    // Imprimir el resultado
    cout << "La potencia de " << A << " sobre " << C << " es: " << valor << endl;

    return 0;
}

File: java/ciclofor.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Solicitar el primer valor
        System.out.print("Ingrese el primer valor: ");
        int A = scanner.nextInt();
        
        int B = 0;

        // Solicitar el segundo valor
        System.out.print("Ingrese el segundo valor: ");
        int C = scanner.nextInt();

        // Calcular la potencia
        int valor = (int) Math.pow(A, C); 
        System.out.println("La potencia de " + A + " sobre " + C + " es: " + valor);

        scanner.close();
    }
}
